### PR TITLE
Origin ARM: Add TVL Adapter

### PR DESCRIPTION
**Pull request by apexearth**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Origin ARM

##### Twitter Link:
https://twitter.com/originprotocol

##### List of audit links if any:
https://docs.originprotocol.com/security-and-risk/audits

##### Website Link:
https://www.originprotocol.com/arm

##### Logo (High resolution, will be shown with rounded borders):
https://www.originprotocol.com/assets/arm_logo-cMnbAcN3.svg

##### Current TVL:
1.96 M

##### Chain:
Ethereum Mainnet

##### Short Description (to be shown on DefiLlama):
The Origin ARM vault allows users to deposit ETH and earn passive yield. The ARM earns yield by using ETH to buy stETH from the market at a discount and redeeming it 1:1 through Lido’s stETH redemption queue. 

##### Token address and ticker if any:
0x85b78aca6deae198fbf201c82daf6ca21942acc6 ARM-WETH-stETH

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yield

##### methodology (what is being counted as tvl, how is tvl being calculated):
WETH, stETH, and redeeming stETH controlled by contract

##### Github org/user (Optional, if your code is open source, we can track activity):
Org: https://github.com/OriginProtocol
Repo: https://github.com/OriginProtocol/arm-oeth

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/originarm exports TVL: 
        
 
 
    ethereum                  1.95 M
    
    total                    1.95 M

File: projects/originarm/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    tvl: async (api) => {
+      const originLidoArm = "0x85b78aca6deae198fbf201c82daf6ca21942acc6";
+      const [weth, steth, outstandingSteth] = await Promise.all([
+        api.call({abi: 'erc20:balanceOf', target: originLidoArm, params: ADDRESSES.ethereum.WETH }),
+        api.call({abi: 'erc20:balanceOf', target: originLidoArm, params: ADDRESSES.ethereum.STETH }),
+        api.call({abi: 'uint256:lidoWithdrawalQueueAmount', target: originLidoArm }),
+      ])
+      api.add(ADDRESSES.ethereum.WETH, weth)
+      api.add(ADDRESSES.ethereum.STETH, steth + outstandingSteth)
+      return api.sumTokens({ owner: originLidoArm, tokens: [ADDRESSES.ethereum.WETH, ADDRESSES.ethereum.STETH] })
```
Comment (g1nt0ki) [inline projects/originarm/index.js:14]: this leads to doublecount

**Comment by g1nt0ki** [comment]
@apexearth thanks for the PR

**Comment by llamatester** [comment]
The adapter at projects/originarm exports TVL: 
        
 
 
    ethereum                  1.92 M
    
    total                    1.92 M

**Comment by apexearth** [comment]
> @apexearth thanks for the PR

Thanks for fixing it up 🙏

**Pull request merged**